Makefile

```diff
@@ -0,0 +1,7 @@
+all:
+	cd ${SUBSTRAIT_JAVA_HOME}/isthmus; ../gradlew shadowJar
+	mkdir -p jars
```

Comment:
These commands are not going to work on Windows.  This is probably ok, for now, but it should maybe be mentioned in the README somewhere.